libs/wallet-ui/src/components/page/page.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon } from '@vegaprotocol/ui-toolkit'
+import { useNavigate } from 'react-router-dom'
+
+export interface PageProps {
+  name: string
+  children: React.ReactElement
+  back?: boolean
```

Comment:
suggestion (non-blocking): i'd call this something else, if it's only for displaying the icon, maybe `withBackIcon`, something less ambiguous (`back: false == 'forward'?`)